src/otx/algo/segmentation/litehrnet.py

```diff
@@ -18,20 +18,362 @@ class LiteHRNet(MMSegCompatibleModel):
     """LiteHRNet Model."""
 
     def __init__(self, num_classes: int, variant: Literal["18", 18, "s", "x"]) -> None:
-        model_name = f"litehrnet_{variant}"
-        config = read_mmconfig(model_name=model_name)
+        self.model_name = f"litehrnet_{variant}"
+        config = read_mmconfig(model_name=self.model_name)
         super().__init__(num_classes=num_classes, config=config)
 
     @property
     def _export_parameters(self) -> dict[str, Any]:
         """Defines parameters required to export a particular model implementation."""
         parent_parameters = super()._export_parameters
         parent_parameters.update(
-            {"onnx_export_configuration": {"operator_export_type": OperatorExportTypes.ONNX_ATEN_FALLBACK}},
+            {
+                "onnx_export_configuration": {"operator_export_type": OperatorExportTypes.ONNX_ATEN_FALLBACK},
+                "via_onnx": True,
+            },
         )
 
         return parent_parameters
 
     def load_from_otx_v1_ckpt(self, state_dict: dict, add_prefix: str = "model.model.") -> dict:
         """Load the previous OTX ckpt according to OTX2.0."""
         return OTXv1Helper.load_seg_lite_hrnet_ckpt(state_dict, add_prefix)
+
+    @property
+    def _optimization_config(self) -> dict[str, Any]:
+        """PTQ config for LiteHRNet."""
+        # TODO(Kirill): check PTQ without adding the whole backbone to ignored_scope #noqa: TD003
+        ignored_scope = self._obtain_ignored_scope()
+        optim_config = {
+            "advanced_parameters": {
+                "activations_range_estimator_params": {
+                    "min": {"statistics_type": "QUANTILE", "aggregator_type": "MIN", "quantile_outlier_prob": 1e-4},
+                    "max": {"statistics_type": "QUANTILE", "aggregator_type": "MAX", "quantile_outlier_prob": 1e-4},
+                },
+            },
+        }
+        optim_config.update(ignored_scope)
+        return optim_config
+
+    def _obtain_ignored_scope(self) -> dict[str, Any]:
```

Comment:
Just a question, why only `litehrnet` needs this complex ignored scope?

Reply:
HRNet contains a huge amount of operations and produces sparse representations, so PTQ fails to process it effectively